torchvision/utils.py

```diff
@@ -223,6 +223,9 @@ def draw_bounding_boxes(
 
     colors = [(ImageColor.getrgb(color) if isinstance(color, str) else color) for color in colors]
 
+    if font is None and font_size != 10:
```

Comment:
I have a minor confusion here. The default argument for `font_size` is set to 10. How do I check if it's the user set or the default value?

cc @oke-aditya @pmeier

Reply:
I would change the default to `None`. With that you can do something like

```py
if font is None and font_size is not None:
    warnings.warn(...)
    font_size = None
elif font is not None and font_size is None:
    font_size = 10
```